pages/activities.jsx

```diff
@@ -277,7 +277,7 @@ var ActivitiesPage = React.createClass({
               <HiveLink
                 src1x="/img/pages/activities/svg/logo-hive-community.svg"
```

Comment:
Oh yea I like that idea :+1: and maybe it's the underlying `<ImageTag>` component (our creation) that we need to make changes to?